src/addReviewServer.js

```diff
@@ -0,0 +1,17 @@
+import fetch from "node-fetch";
+
+const API_BASE = "https://plankton-app-xhkom.ondigitalocean.app/api/reviews";
+
+export async function postReview(req, res) {
+  const method = {
+    method: "POST",
+    body: JSON.stringify(req.body),
+    headers: {
+      Accept: "application/json",
+      "Content-Type": "application/json",
+    },
+  };
+  // i am going to use this variable in the second part of the assignment.
+  const response = await fetch(API_BASE, method);
```

Comment:
Even if you plan on using the variable later, I would recommend against committing unused variables, because circumstances might change and you might have to focus your effort elsewhere, leaving this unused variable here to confuse others.

You can always just add it later, when you actually need it.